Install tarball links both point to non-bundled archive

## Bug Report

I am running a custom LFS based distro and I downloaded crystal lang from the download page as a bundle:

https://github.com/crystal-lang/crystal/releases/download/1.14.0/crystal-1.14.0-1-linux-x86_64.tar.gz

I unpacked the tar ball under /usr/local 

```
# crystal -v
Crystal 1.14.0 [dacd97bcc] (2024-10-09)

LLVM: 18.1.6
Default target: x86_64-unknown-linux-gnu
```

I see that /usr/local/lib/crystal contains a single file `libgc.a` 

My program:

```
# cat src/cr.cr 
# TODO: Write documentation for `Cr`
module Cr
  VERSION = "0.1.0"

  # TODO: Put your code here
  puts "Hello World!"
end
```

I tried to run it:

```
# crystal run src/cr.cr
/usr/bin/ld: cannot find -levent (this usually means you need to install the development package for libevent): No such file or directory
collect2: error: ld returned 1 exit status
Error: execution of command failed with exit status 1: cc "${@}" -o /root/.cache/crystal/crystal-run-cr.tmp  -rdynamic -L/usr/local/bin/../lib/crystal -lgc -lpthread -levent -lrt -lpthread -ldl
```

This is due to `libevent.a` missing in `/usr/local/lib/crystal`.  Is it expected to be on the host?  In what sense is the download a `bundle`?

I assumed that a download would contain its own dependencies to be able to generate ready-to-run binaries.  `ld` is fairly fundamental but so far I have installed +200 packages without needing `libevent` in any form or shape so its not indispensable.

--- a/_data/packages.yaml
+++ b/_data/packages.yaml
@@ -19,7 +19,7 @@
   arch: [x86_64, x86_64-bundled]
   title: Tarball (`.tar.gz`)
   instructions_href: /install/from_targz/
-  archive_href: https://github.com/crystal-lang/crystal/releases/download/{{ version }}/crystal-{{ version }}-1-linux-x86_64.tar.gz
+  archive_href: https://github.com/crystal-lang/crystal/releases/download/{{ version }}/crystal-{{ version }}-1-linux-{{ arch }}.tar.gz
   repology: crystal
 
 - type: system

